crates/bevy_pbr/src/render/mesh.rs

```diff
@@ -741,10 +749,13 @@ impl SpecializedMeshPipeline for MeshPipeline {
             }
         }
 
-        let format = if key.contains(MeshPipelineKey::HDR) {
-            ViewTarget::TEXTURE_FORMAT_HDR
-        } else {
-            TextureFormat::bevy_default()
+        if key.contains(MeshPipelineKey::ENVIRONMENT_MAP) {
+            shader_defs.push("ENVIRONMENT_MAP".into());
+        }
+
+        let format = match key.contains(MeshPipelineKey::HDR) {
+            true => ViewTarget::TEXTURE_FORMAT_HDR,
+            false => TextureFormat::bevy_default(),
         };
```

Comment:
why was this changed?

Reply:
I thought it looked better, it keeps the two texture formats right next to each other instead of separated like an if statement. I can revert it though.

Reply:
While you aren't the only one that likes it, cart doesn't and has asked to not use it in the past. I thought we had a clippy rule for that. I don't really mind personally.

Reply:
Reverted.